packages/dds/task-manager/src/test/taskManager.fuzz.spec.ts

```diff
@@ -264,13 +268,17 @@ describe("TaskManager fuzz testing with rebasing", () => {
 	createDDSFuzzSuite(model, {
 		validationStrategy: { type: "fixedInterval", interval: defaultOptions.validateInterval },
 		// AB#5185: enabling rebasing indicates some unknown eventual consistency issue
-		skip: [0, 2, 6],
+		skip: [5, 7],
```

Comment:
I'm not familiar with this suite generation, but does this mean we're now skipping a different set of tests? Are they new failures?

Also the comment above mentions AB#5185 - is that work item still valid after this change?